lib/livebook/config.ex

```diff
@@ -184,6 +184,18 @@ defmodule Livebook.Config do
     not match?({LivebookWeb.SessionIdentity, _}, Livebook.Config.identity_provider())
   end
 
+  @doc """
+  Returns identity source as a friendly atom.
+  """
+  @spec identity_source() :: atom()
+  def identity_source() do
+    case identity_provider() do
+      {Livebook.ZTA.GoogleIAP, _} -> :google_iap
+      {Livebook.ZTA.Cloudflare, _} -> :cloudflare
+      {LivebookWeb.SessionIdentity, _} -> :session
+    end
```

Comment:
Can we perhaps do something to keep this in sync with this: https://github.com/livebook-dev/livebook/blob/de41f0a58ffedc4a6eea6ec7296bda454d562d18/lib/livebook/config.ex#L528 ?

For example, we are using different names for `googleiap`. Maybe we can have two module attributes from %{module => atom} and another from %{string => module} that we generate based on the same input data?

Reply:
Good call!